backend/catalogue/management/commands/generate_layer_previews.py

```diff
@@ -78,42 +92,58 @@ def retrieve_image(layer):
 class Command(BaseCommand):
     def add_arguments(self, parser):
         parser.add_argument(
-            '--layer_id'
+            '--layer_id',
+            help='Specify a layer ID for the layer preview image you want to generate'
+        )
+        parser.add_argument(
+            '--only_generate_missing',
+            help='If true, skips generating images for layers where images already exist'
         )
 
     def handle(self, *args, **options):
-        layer_id = int(options['layer_id']
-                       ) if options['layer_id'] != None else None
+        layer_id = int(options['layer_id']) if options['layer_id'] != None else None
+        only_generate_missing = options['only_generate_missing'].lower() in ['t', 'true'] if options['only_generate_missing'] != None else False
+
```

Comment:
Trivial, but there's a tonne of whitespace here!

Reply:
Fixed!